.travis.yml

```diff
@@ -53,8 +44,12 @@ env:
   - TARGET=":jsettlers.logic:test:aiDifficultiesIT"
   - TARGET=":jsettlers.logic:test:autoReplayIT"
   - TARGET=":jsettlers.logic:test:replayValidationIT"
-    
-  # Test release generation
+  
   - TARGET="release"
 
+  
 script: GRADLE_OPTS='-Xmx600m -Dorg.gradle.jvmargs="-Xmx1500m"' ./gradlew $TARGET
+
+
+after_success:
+- ./deploy.sh
```

Comment:
No, `after_success` is only run once after all the matrix elements have successfully been build. What you mean is `after_script`.

See https://docs.travis-ci.com/user/customizing-the-build/